README.md

```diff
@@ -871,7 +871,7 @@ $ cat /usr/share/ramalama/shortnames.conf
     </summary>
     <br>
 
-    You can `pull` a model using the `pull` command. By default, it pulls from the Ollama registry.
+    You can `pull` a model using the `pull` command. By default, it pulls from the <a href="https://ollama.com/library">Ollama registry</a>.
```

Comment:
This is not proper markdown.

[Ollama registry]("https://ollama.com/library").